packages/passport-interface/src/SubscriptionManager.ts

```diff
@@ -55,25 +55,26 @@ export class FeedSubscriptionManager {
   }
 
   public async pollSubscriptions(): Promise<SubscriptionActions[]> {
-    const responses: SubscriptionActions[] = [];
-
-    for (const subscription of this.activeSubscriptions) {
-      try {
-        responses.push({
-          actions: (
-            await this.api.pollFeed(subscription.providerUrl, {
-              feedId: subscription.feed.id,
-              pcd: subscription.credential
-            })
-          ).actions,
-          subscription
-        });
-      } catch (e) {
-        console.log(`failed to poll subscription`, e);
-      }
-    }
+    const results = await Promise.all(
```

Comment:
if any feed fails here, that will cause `Promise.all` to reject, which is not the behavior we want. You should instead use `Promise.allSettled`, and then go through each result, and react to each promise result appropriately.